doc/benchmark.rst

```diff
@@ -0,0 +1,21 @@
+Benchmark different kernel implementations
+##########################################
+
+You can benchmark different gate implementations inside the project.
+
+.. code-block:: console
+
+    $ make benchmark
+    $ cd BuildBench
+
+Inside the directory, you can see ``run_gate_benchmark.sh``. You can benchmark a specific kernel implementation for a gate by providing arguments. 
+For example, you can compare benchmark results for ``PauliX`` gate with ``KernelType::PI`` and ``KernelType::LM`` by running the following commands:
```

Comment:
Do we discuss the differences between `LM` and `PI` gates anywhere in the docs? Maybe I haven't got there yet.

Reply:
I add the results.

Reply:
I updated the docs. I also uploaded my benchmark results in https://github.com/PennyLaneAI/pennylane-lightning-compare-kernels.

Reply:
Nice work!